inc/invitation.class.php

```diff
@@ -78,6 +78,9 @@ public function getRights($interface = 'central') {
       return $rights;
    }
 
+   /**
+    * @see CommonDBTM::prepareInputForAdd()
```

Comment:
I added myself many @see annotations, but this is not a good idea. Drop them instead.

See the guidelines from GLPI to know why
http://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#override-method-inheritdoc-see-docblock-no-docblock